bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/sitequality/SiteQualityPanel.java

```diff
@@ -338,53 +365,27 @@ private static String formatPeriod(final TimingWindow tw) {
 		return String.format("%02d:%02d:%02d", hh, mm, ss);
 	}
 
-	private static String formatRepeat(TimingWindow tw) {
+	private static String formatRepeat(final TimingWindow tw) {
 		if (tw.getDuration() == TimingWindow.WINDOW_REMAINS_OPEN_FOREVER) return null;
 		switch (tw.getRepeat()) {
-		case -1: return "forever";
-		case 0: return "never";
-		case 1: return "1 time";
-		default: return tw.getRepeat() + " times";
+			case -1: return "forever";
```

Comment:
You're right. This is kind of stupid, since there are constants already defined in the `TimingWindow` class precisely for this. I'll change -1 and 0 to use these.